core/src/impl/Kokkos_Profiling.hpp

```diff
@@ -515,7 +520,12 @@ void beginDeepCopy(const SpaceHandle dst_space, const std::string dst_label,
                    const uint64_t size);
 void endDeepCopy();
 void finalize();
-void initialize();
+void initialize(const std::string& = {});
```

Comment:
Wasn't this forward declaration a duplicate?

Reply:
No, there is a `Kokkos::Profiling::initialize(...)` and a `Kokkos::Tools::initialize(...)`